ui/src/components/helpers/WelcomeTutorialFunctions.ts

```diff
@@ -0,0 +1,33 @@
+const tutorialKey = "tutorial_timestamp";
+
+function setTimeStamp() {
```

Comment:
Jest won't import it when I set return type 😞 `function setTimeStamp(): void{
`

Reply:
Oh hmm it's cause jest is not parsing typescript. I'll take a look tomorrow to see what's up, weird.